solutions/devsprint-bruno-faganello-1/FinanceApp/Components/ActivityListView.swift

```diff
@@ -67,13 +73,13 @@ extension ActivityListView {
 extension ActivityListView: UITableViewDataSource {
 
     public func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
-
-        return 5
+        return items.count
     }
 
     public func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
 
         let cell = tableView.dequeueReusableCell(withIdentifier: cellIdentifier, for: indexPath) as! ActivityCellView
+        cell.setupWithActivity(items[indexPath.row])
```

Comment:
[REQUEST CHANGED]
Verifica se o array tem o elemento...pq pode acontecer de dar um índex out of range